invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/invitationsModal/GroupTabPane.js

```diff
@@ -112,12 +112,11 @@ export class GroupTabPane extends Component {
                 onOptionChangeCallback={this.handleRoleUpdate}
               />
             </Form.Field>
-            <Form.Field disabled>
-              <>
-                <label>{i18next.t("Message")}</label>
-                <RichEditor />
-              </>
-            </Form.Field>
+            <i>
+              {i18next.t(
+                "Note: upon addition, selected groups will become community members immediately without any kind of notification."
```

Comment:
minor:
```suggestion
                "Note: upon addition, selected groups will become community members immediately without any kind of notification or invitation approval."
```

Reply:
Thanks for noticing!
I'm not sure the confirmation modal is necessary here, as we are not doing something dangerous that can not be undone. After closing the modal, user can immediately remove groups access if they did something wrong. What do you think?